CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="3.3.0"></a>
+
+## [3.3.0] - 2023-04-26
```

Comment:
Since we have renamed the method, this is considered a breaking change. major version upgrade.